includes/class-wc-payments-account.php

```diff
@@ -73,6 +73,7 @@ public function __construct( WC_Payments_API_Client $payments_api_client, Databa
 		add_filter( 'allowed_redirect_hosts', [ $this, 'allowed_redirect_hosts' ] );
 		add_action( 'jetpack_site_registered', [ $this, 'clear_cache' ] );
 		add_action( 'updated_option', [ $this, 'possibly_update_wcpay_account_locale' ], 10, 3 );
+		add_action( 'woocommerce_woocommerce_payments_updated', [ $this, 'clear_cache' ] );
```

Comment:
This will clear the account cache for every version that is greater than the current version update.  @c-shultz, is this expected or do we want to do this on specific versions?